test/onmt/ExtendedCmdLineTest.lua

```diff
@@ -100,8 +100,11 @@ end
 function cmdLineTest.fail_unknown()
   local cmd = onmt.utils.ExtendedCmdLine.new()
   onmt.Seq2Seq.declareOpts(cmd)
-
+  onmt.data.SampledDataset.declareOpts(cmd)
+  -- will generate error
+  print('')
```

Comment:
Yes but with e794053, `ExtendedCmdLine` no more writes on stderr when used inside the test suite. It throws an actual Lua error which is correctly handled by the `tester:assertError()` method.